tests/Feature/Endpoints/ShipmentsTest.php

```diff
@@ -80,6 +80,49 @@ public function create_a_new_shipment_concept_for_a_parcel()
         $this->assertTrue($this->cleanUp($shipment));
     }
 
+    /** @test */
+    public function create_a_new_shipment_concept_for_a_parcel_with_insurance()
+    {
+        $array = [
+            'reference_identifier' => 'test-123',
+            'recipient' => [
+                'company' => 'Test Company B.V.',
+                'first_name' => 'John',
+                'last_name' => 'Doe',
+                'email' => 'john@example.com',
+                'phone' => '0101111111',
+                'street' => 'Poststraat',
+                'number' => '1',
+                'number_suffix' => 'A',
+                'postal_code' => '1234AA',
+                'city' => 'Amsterdam',
+                'region' => 'Noord-Holland',
+                'cc' => 'NL',
+            ],
+            'options' => [
+                'label_description' => 'Test label description',
+                'large_format' => false,
+                'only_recipient' => false,
+                'package_type' => PackageType::PACKAGE,
+                'return' => false,
+                'signature' => true,
+            ],
+        ];
+
+        $parcel = new Parcel($array);
+        $parcel->insurance(500);
```

Comment:
You are calling the new `insurance` method here, however there is no assertion in the tests that verifies this.

Reply:
Added assertion